test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -394,7 +395,8 @@ class ClientLbEnd2endTest : public ::testing::Test {
     const int port_;
     std::unique_ptr<Server> server_;
     MyTestServiceImpl service_;
-    experimental::OrcaService orca_service_;
+    experimental::ServerMetricRecorder server_metric_recorder_;
+    std::unique_ptr<experimental::OrcaService> orca_service_;
```

Comment:
Why did you make this a `unique_ptr<>`?  I think it was fine being a direct data member.

Reply:
Still need to address this.

Reply:
Done.